antora-playbook.yml

```diff
@@ -77,48 +91,48 @@ content:
   - url: https://github.com/couchbase/docs-connectors-talend
   # - url: https://git@github.com/couchbase/docs-analytics
   - url: https://github.com/couchbase/docs-analytics
-    branches: [release/7.2, release/7.1, release/7.0]
+    branches: [release/7.6, release/7.2, release/7.1, release/7.0]
   - url: https://github.com/couchbase/couchbase-cli
-    branches: [neo, 7.1.x-docs, cheshire-cat]
+    branches: [trinity, neo, 7.1.x-docs, cheshire-cat]
     start_path: docs
   # - url: https://git@github.com/couchbase/backup
   - url: https://github.com/couchbase/backup
-    branches: [neo, 7.1.x-docs, cheshire-cat]
+    branches: [trinity, neo, 7.1.x-docs, cheshire-cat]
     start_path: docs
   # NOTE docs-server is currently after other server repos so nav key wins
   - url: https://github.com/couchbase/docs-server
-    branches: [release/7.2, release/7.1, release/7.0]
+    branches: [release/7.6, release/7.2, release/7.1, release/7.0]
   - url: https://github.com/couchbase/docs-sdk-common
-    branches: [release/7.2, release/7.1.2, release/7.1, release/7.0, release/6.6, release/6.5]
+    branches: [temp/7.5, release/7.2, release/7.1.2, release/7.1, release/7.0, release/6.6, release/6.5]
   - url: https://github.com/couchbase/docs-sdk-c
-    branches: [release/3.3, release/3.2]
+    branches: [release/3.3]
   - url: https://github.com/couchbase/docs-txn-cxx
     branches: [release/1.0]
   - url: https://github.com/couchbase/docs-sdk-dotnet
-    branches: [release/3.4, release/3.3, release/3.2]
+    branches: [release/3.5, release/3.4, release/3.3]
```

Comment:
These are all one dotminor behind, and match 7.6, rather than 7.6.2 - is that deliberate, or should they be updated?

Reply:
Take off and nuke the site from orbit